public/app/features/correlations/CorrelationsPage.test.tsx

```diff
@@ -90,8 +98,17 @@ const renderWithContext = async (
     },
   } as unknown as BackendSrv;
   const grafanaContext = getGrafanaContextMock({ backend });
-  setBackendSrv(backend);
-  setupDataSources(...Object.values(datasources));
+  const dsServer = new MockDataSourceSrv(datasources) as unknown as DataSourceSrv;
```

Comment:
We can't be using `setupDataSources` as it relies on actual import of the module that we add in https://github.com/grafana/grafana/blob/main/public/app/features/plugins/built_in_plugins.ts. Therefore we need use a mock instead. 

Btw this is [based on this PR and original test](https://github.com/grafana/grafana/pull/97748/files) mock before it was (recently) replaced with setupDataSources